src/scripts/main.js

```diff
@@ -1,8 +1,24 @@
 'use strict';
 
 const people = require('./lib/people');
+const tableBody = document.querySelector('tbody');
```

Comment:
```suggestion
const dashboard = document.querySelector('.dashboard');
```
I recommend you be more specific.